apps/builder/app/builder/features/breakpoints/breakpoints-editor.tsx

```diff
@@ -1,95 +1,145 @@
-import { useState } from "react";
```

Comment:
(Not related to line, just to create a thread)

I added a breakpoint. After reloading page, the order is different in list vs editor:


<img width="258" alt="Screenshot 2023-05-01 at 14 34 57" src="https://user-images.githubusercontent.com/825702/235446381-220156dd-e029-480e-b748-ddbed4b2109e.png">

<img width="258" alt="Screenshot 2023-05-01 at 14 36 05" src="https://user-images.githubusercontent.com/825702/235446499-87cf0cf5-a8c0-4a6d-a4fe-e7204f45ce15.png">

Reply:
Ah, also double separator in above "Done" in the screenshot above looks strange

Reply:
order is expected, its going to be sorted after closing too

Reply:
actually the order is incorrect, after closing and opening it should be the same order as in the other lists

Reply:
> its going to be sorted after closing too

After closing the popover? For me it stays different after closing, reloading page, and opening again